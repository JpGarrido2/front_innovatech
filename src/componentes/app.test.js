import React from "react";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { shallow } from "enzyme";
import App from "./app";

describe("Mi App", () => {
  it("debe ejecutar", () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });
});
