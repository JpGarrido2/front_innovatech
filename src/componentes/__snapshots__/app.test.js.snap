// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mi App debe ejecutar 1`] = `
<div
  className="container-fluid px-0 h-100"
>
  <AutenticancionProveedor>
    <Encabezado />
    <Routes>
      <Route
        element={<PaginaInicio />}
        path="/"
      />
      <Route
        element={
          <RequerirAutenticacion
            redirectTo="/"
          >
            <Avances />
          </RequerirAutenticacion>
        }
        path="/avances"
      />
      <Route
        element={
          <RequerirAutenticacion
            redirectTo="/"
          >
            <Administracion />
          </RequerirAutenticacion>
        }
        path="/proyectos"
      />
      <Route
        element={
          <RequerirAutenticacion
            redirectTo="/"
          >
            <Administracion />
          </RequerirAutenticacion>
        }
        path="/estudiantes"
      />
      <Route
        element={<PaginaAcercaDe />}
        path="/acercade"
      />
      <Route
        element={<PaginaContacto />}
        path="/contacto"
      />
      <Route
        element={<PaginaNoEncontrada />}
        path="*"
      />
    </Routes>
    <Pie />
  </AutenticancionProveedor>
</div>
`;
